README.md

```diff
@@ -38,29 +38,46 @@ open data.
+The default exported image contains all the profiles, but a custom image can be created with only the needed
+dependencies.
+
+| Profile                    | Dependencies                                           | Description                                                                                                                 |
+|----------------------------|--------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| `fragmentation`            | `postgres-pagination-repository`                       | Allows basic fragmentation (pagination)                                                                                     |
+| `maintenance`              | `postgres-maintenance-repository`                      | Allows the LDES server to perform maintenance operations on its Event Streams and Members: compaction, retention, deletion. |
+| **Interfaces**             |                                                        |                                                                                                                             |
+| `http-admin`               | `ldes-server-admin-rest`,`postgres-admin-repository`** | Gives access to REST API to create and manage Event Streams and Views.                                                      |
+| `http-ingest`              | `ldes-server-ingest-rest`,`postgres-ingest-repository` | Gives access to REST API to ingest members into the LDES.                                                                   |
+| `http-fetch`               | `ldes-server-fetch-rest`,`postgres-fetch-repository`   | Gives access to REST API to fetch Event Streams, its Views and pages.                                                       |
+| **Plugin Fragmentations**  |                                                        |                                                                                                                             |
+| `fragmentation-timebased`  | `ldes-server-fragmentation-timebased-hierarchical`     | Allows fragmentation in based on a timebased property.                                                                      |
+| `fragmentation-geospatial` | `ldes-server-fragmentation-geospatial`                 | Allows fragmentation in based on a geospatial property.                                                                     |
+| `fragmentation-reference`  | `ldes-server-fragmentation-reference`                  | Allows fragmentation in based on a textual property.                                                                        |
+
+*: The `ldes-server-domain` is the core domain module of the LDES server and is always loaded in.
+To keep the graph clean, it is not shown in the graph.
+Next to that, the `postgres-liquibase` module will always be loaded in to assure correct database creation.
```

Comment:
Why not putting it here and not directly below the graph?

Reply:
just to have the stars somewhat together

Reply:
I meant more like why putting it there as a star and not like "regular" content below the graph?